client/vendor-dll.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Dependencies we would like to like in the vendor chunk. These can be external or internal deps, but either way,
+ * they should be things that the majority of the application uses, and things that don't change very often.
+ */
+require( 'classnames' );
+require( 'debug' );
+require( 'i18n-calypso' );
+// lodash would go here, but we leave it out because we use tree shaking to only require the bits that we use.
```

Comment:
it's 4kb for the [core build](https://github.com/lodash/lodash/wiki/build-differences), which doesn't include everything we use. The full build is around 23k after gzip.

It's a good question though. We might save more by pushing all of lodash into vendor and not tree shaking at all, since we're duping some parts of lodash across chunks.